Start of project1/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <figure>
+        <img src="images/octopus092118.jpg" alt="Lalala">
+        <figcation>
+
+        </figcation>
+    </figure>
+    <h1>Yes, the Octopus Is Smart as Heck. But Why? </h1>
+    <p>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+        intelligent.</p>
+
+
+    <p><img src="https://static01.nyt.com/images/2018/12/04/science/04SCI-ZIMMER/merlin_147494628_a788760f-4209-40db-a0be-1305c3a661a6-superJumbo.jpg?quality=90&auto=web"
+            width="650" height="400">
+
+        <p>An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchers are
+            uncertain how the trait evolved</p>
+
+        <p> Fred Tanneau/Agence France-Presse — Getty Images</p>
+
+        <p>
```

Comment:
You don't want to wrap an `h2` (or any of the heading elements) in a `p` element:
```suggestion

```